config/php/docker-entrypoint.sh

```diff
@@ -62,6 +62,32 @@ function waitUntil() {
   done
 }
 
+function setPermissions()
+{
+  mkdir -p var/cache var/log public/multimedia public/thumbnail public/avatar import export
+  >&2 echo "Setting file permissions..."
+  if setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX var ; then
```

Comment:
Could we not go recursive on condition?
It'd generate **a lot** of error logs about every single file in dir not being supported and therefore could lead to confusion whether everything is correct.
Alternatively, we could verify fs itself with sth like `df`.
Up to you